index.html

```diff
@@ -5,6 +5,10 @@
     <title>Plumber</title>
     <link rel="stylesheet" type="text/css" href="style.css">
     <meta charset="UTF-8">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
Please make sure all the paths you've added are correct. This is the correct one: `./favicon/apple-touch-icon.png`.
Also, use the same approach with `./` for `style.css`.